commands/gen.go

```diff
@@ -63,7 +63,7 @@ boolean flags to disabled security features. Examples:
 
 func init() {
 	rootCmd.AddCommand(genCmd)
-
+	genCmd.Flags().StringVarP(&opts.Namespace, "namespace", "n", "", "Kubernetes namespace to use.")
```

Comment:
```suggestion
	genCmd.Flags().StringVarP(&opts.Namespace, "namespace", "n", "", "Kubernetes namespace to use")
```
Sorry for this nit, it's just to be uniform with the rest!

No problem I will squash when merging :)